WordPress/Classes/ViewRelated/Comments/EditCommentTableViewController.swift

```diff
@@ -47,8 +61,26 @@ class EditCommentTableViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        // TODO: return custom cell
-        return UITableViewCell()
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: EditCommentSingleLineCell.defaultReuseID) as? EditCommentSingleLineCell else {
+            return UITableViewCell()
+        }
+
+        switch indexPath.section {
+        case 0:
```

Comment:
It just occurred to me I should have made these sections constants for clarity. I will do that in the next PR (I'm changing this anyway).